pipeline/scripts/generate_truth_data.jl

```diff
@@ -16,14 +16,6 @@ include(scriptsdir("common_param_values.jl"));
 ## Set up the truth Rt and save a plot of it
 
 using .AnalysisPipeline, Plots, JLD2
```

Comment:
why does this still exist or converly why did it get changed? It seems like having a script that generates and saves this is a good idea? 

I am getting concerned about the lack of check pointing in this workflow FYI.